README.md

```diff
@@ -630,6 +651,34 @@ userRequester
     .then(process.exit);
 ```
 
+Example with `async / await`
+
+`responder.js`
+```js
+const cote = require('cote');
+
+const sumResponder = new cote.Responder({ name: 'Sum Responder' });
+
+// Responder event needs to return a promise, so that requester 
+//will resolve / reject. That is why we make our function `async`
+userResponder.on('sum', async (req) => req.a + req.b);
+```
+
+`requester.js`
+```js
+const cote = require('cote');
+const sumRequester = new cote.Requester({ name: 'Sum Requester' });
+
+async function requestSum() {
+  const sum = await userRequester.send({ type:'sum', a:3, b:5 });
```

Comment:
It would be good to use the exact example in promises above, instead of a sum operation.